RocksmithTookitGUI/DLCPackageCreator/DLCPackageCreator.Designer.cs

```diff
@@ -861,15 +837,50 @@ private void InitializeComponent()
             this.label4.TabIndex = 14;
             this.label4.Text = "dB";
             // 
+            // label9
+            // 
+            this.label9.AutoSize = true;
+            this.label9.Font = new System.Drawing.Font("Microsoft Sans Serif", 6.25F, System.Drawing.FontStyle.Regular, System.Drawing.GraphicsUnit.Point, ((byte)(204)));
+            this.label9.ForeColor = System.Drawing.Color.MediumSlateBlue;
+            this.label9.Location = new System.Drawing.Point(139, 92);
+            this.label9.Name = "label9";
+            this.label9.Size = new System.Drawing.Size(32, 12);
+            this.label9.TabIndex = 21;
+            this.label9.Text = "EN/JP";
+            // 
+            // txtArtist
+            // 
+            this.txtArtist.BackColor = System.Drawing.SystemColors.Window;
+            this.txtArtist.Cue = "Artist";
+            this.txtArtist.Font = new System.Drawing.Font("Microsoft Sans Serif", 8.25F);
+            this.txtArtist.ForeColor = System.Drawing.Color.Gray;
+            this.txtArtist.Location = new System.Drawing.Point(8, 17);
+            this.txtArtist.Name = "txtArtist";
+            this.txtArtist.Size = new System.Drawing.Size(154, 20);
+            this.txtArtist.TabIndex = 0;
+            this.toolTip.SetToolTip(this.txtArtist, "Artist");
+            // 
+            // txtSongTitle
+            // 
+            this.txtSongTitle.BackColor = System.Drawing.SystemColors.Window;
```

Comment:
I'd try to get lesser changes here in future since designer is dangerous place to thinker thanks to visual studio routines.